Readme.txt

```diff
@@ -0,0 +1,14 @@
+List of Patterns Identified in the Current Scenario
+Creational Pattern
+Abstract Factory

```

Comment:
How would you implement Factory and Builder? Where would you use Builder, and where Abstract Factory?